lib/spack/env/cc

```diff
@@ -33,6 +33,9 @@ parameters=(
     SPACK_CXX_RPATH_ARG
     SPACK_F77_RPATH_ARG
     SPACK_FC_RPATH_ARG
+    SPACK_DTAGS_TO_ENABLE
```

Comment:
Can we change these to `DTAGS_TO_STRIP` and `DTAGS_TO_ADD`? `enable|disable` is easy to get confused with `--enable-new-dtags` and `--disable-new-dtags`, so I think the code will be more readable with the change.

Reply:
I see that Todd has requested changes that would make this irrelevant.

Reply:
@becker33 See https://github.com/spack/spack/pull/9168#discussion_r217381411  and https://github.com/spack/spack/pull/9168#issuecomment-421707522 I'd prefer to generalize the mechanism we use to handle injecting and stripping flags in a following PR. If that's fine with everybody I'll do the renaming you propose above.

Reply:
Done in f354176